w5e1.py

```diff
@@ -0,0 +1,82 @@
+"w5e1" 
+import random
+import os
+
+# pokemon s1e1 -> Ash late to professor Oak so can't choose pokemon...
```

Comment:
You dont need this comment here - just give an indicative filename

Reply:
Delete the comment